src/plugins/csp_handler/server/csp_handlers.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ConfigurationClient } from '../../application_config/server';
+import {
+  CoreSetup,
+  IScopedClusterClient,
+  Logger,
+  OnPreResponseHandler,
+  OnPreResponseInfo,
+  OnPreResponseToolkit,
+  OpenSearchDashboardsRequest,
+} from '../../../core/server';
+
+const CSP_RULES_CONFIG_KEY = 'csp.rules';
+
+export function createCspRulesPreResponseHandler(
```

Comment:
it is worth to add description/jsdoc for this method

Reply:
Added